src/njoy21/input/ACER/Card6/Ismooth/test/Ismooth.test.cpp

```diff
@@ -0,0 +1,46 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "Ismooth  output values",
+         "[ACER],[Card6], [Ismooth]"){
+
+  long ln{0};
+  GIVEN( "valid ismooth values" ){
+    std::vector<int> validValues{0, 1};
+
+    THEN( "the returned class has the correct value" ){
+      for( auto& ismooth : validValues ){
```

Comment:
@nicholas-sly has discovered that this `for` loop needs to be outside the `THEN` and the string passed to the `THEN` needs to be unique or `Catch++` will assume it's the exact same test.